### PR TITLE
cosmetic changes

### PR DESCRIPTION
**Pull request by jasonrichardsmith**

### CODE REVIEW

File: data/about.yml [hunk 1]
```diff
@@ -3,13 +3,13 @@ heading : Our
 headingSpan : Services
 aboutItem :
 - icon : tf-tools
-  title : Cloud Native consultancy
-  content : We're consulting companies in transition to cloud and agile infrastructure.
+  title : Cloud Native Consultancy
+  content : We consult companies on their cloud native transition, agile infrastructure, and DevOps best practices.
```
Comment (moredhel) [inline data/about.yml:7]: Looks good, maybe ‘we consult with companies...’

File: data/service.yml [hunk 2]
```diff
@@ -3,13 +3,13 @@ heading : Our
 headingSpan : Services
 serviceItem :
 - icon : tf-tools
-  title : Cloud Native consultancy
-  content : We're consulting companies in transition to cloud and agile infrastructure.
+  title : Cloud Native Consultancy
+  content : We consult companies on their cloud native transition, agile infrastructure, and DevOps best practices.
```
Comment (moredhel) [inline data/service.yml:7]: Same as above, what do you think?

File: data/about.yml (hunk 1 above)
Comment (jasonrichardsmith) [reply data/about.yml]: done

File: data/service.yml (hunk 2 above)
Comment (jasonrichardsmith) [reply data/service.yml]: done

**Pull request merged**